pkg/kubelet/cm/dra/manager.go

```diff
@@ -134,27 +137,42 @@ func (m *ManagerImpl) PrepareResources(pod *v1.Pod) error {
 			if pluginName == "" {
 				pluginName = resourceClaim.Status.DriverName
 			}
+			batches[pluginName] = append(batches[pluginName],
+				&drapb.Claim{
+					Namespace:      resourceClaim.Namespace,
+					Uid:            string(resourceClaim.UID),
+					Name:           resourceClaim.Name,
+					ResourceHandle: resourceHandle.Data,
+				})
```

Comment:
nit: but I'd prefer to split this apart, i.e.:

```
pbclaim := &drapb.Claim{
	Namespace:      resourceClaim.Namespace,
	Uid:            string(resourceClaim.UID),
	Name:           resourceClaim.Name,
	ResourceHandle: resourceHandle.Data,
}

batches[pluginName] = append(batches[pluginName], pbclaim)
```

Reply:
Okay, changed.

Reply:
Did this change get reverted as well?

Reply:
This still doesn't seem to be changed...